content/docs/5.1/concepts/tikv-in-5-minutes.md

```diff
@@ -0,0 +1,225 @@
+---
+title: TiKV in 5 Minutes
+description: Get started using TiKV in 5 Minutes
+menu:
+    "5.1":
+        parent: Get Started
+        weight: 1
+---
+
+## Prerequisites
+
+Ensure that you are using MacOS or Linux.
+
+{{< warning >}}
+The following steps are only for test environments.
+{{< /warning >}}
+<!-- TODO: For production environments please refer to ??? -->
+
+## Step 1: Install a Playground TiKV Cluster
+
+1. Install TiUP by executing the following command:
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
+```
+
+2. Set the TiUP environment variables:
+
+Redeclare the global environment variables:
+
+```bash
+source .bash_profile
+```
+
+Confirm whether TiUP is installed:
+```bash
+tiup
+```
+
+3. If TiUP is already installed, update the TiUP playground component to the latest version:
+```bash
+tiup update --self && tiup update playground
+```
+
+4. Use TiUP playground to start a local TiKV cluster
+```bash
+tiup playground --mode tikv-slim
+```
+
+5. Press `Ctrl + C` to stop the local TiKV cluster
+
+{{< info >}}
+Refer to [TiUP playground document](https://docs.pingcap.com/tidb/stable/tiup-playground) to find more TiUP playground commands.
+{{< /info >}}
+
+## Step 2: Monitor the cluster
+
+Open [http://127.0.0.1:3000](http://127.0.0.1:3000) and login to the Grafana (username: admin password: admin).
+
+Open `TiKV-Summary` page on Grafana to find the metrics of your TiKV cluster.
+
+## Step 3: Hello, World!
+
+### Python
+
+1. Install `tikv-client` python package
+
+```bash
+pip3 install -i https://test.pypi.org/simple/ tikv-client
+```
+
+{{< info >}}
+This package requires Python 3.5+.
+{{< /info >}}
+
+2. Try `RAW KV` API
+
+Write test code
+
+```bash
+echo '
+from tikv_client import RawClient
+
+client = RawClient.connect("127.0.0.1:2379")
+
+# put
+client.put(b"k1", b"Hello")
+client.put(b"k2", b",")
+client.put(b"k3", b"World")
+client.put(b"k4", b"!")
+client.put(b"k5", b"Raw KV")
+
+# get
+print(client.get(b"k1"))
+
+# batch get
+print(client.batch_get([b"k1", b"k3"]))
+
+# scan
+print(client.scan(b"k1", end=b"k5", limit=10, include_start=True, include_end=True))
+' > test_txn.py
```

Comment:
would it be more helpful to wrap it in a python code block and ask users to execute it?

Reply:
BTW, the file name should be `test_raw.py` instead of `test_txn.py`

Reply:
nice~